mods/cnc/weapons/explosions.yaml

```diff
@@ -99,3 +99,17 @@ Napalm.Crate:
 		ImpactSounds: flamer2.aud
 	Warhead@3Smu: LeaveSmudge
 		SmudgeType: Scorch
+
+TiberiumExplosion:
+	Inherits: ^DamagingExplosion
```

Comment:
You probably want to inherit `^DamagingExplosionHE` instead. Otherwise you're changing the `Versus` values.